lib/versioning/docker/index.js

```diff
@@ -1,8 +1,96 @@
-const semver = require('../semver');
+function parse(version) {
+  const [prefix, suffix] = version.split('-');
```

Comment:
How does this suffix parsing interact with the datasource suffix parsing? I think datasource returns suffix in the case that the existing version has no suffix.

Reply:
suffix is simply ignored in all operations, at least for now.